docs/InstallGuide.md

```diff
@@ -116,12 +146,9 @@ Modify the following section in /etc/openvnet/common.conf on all nodes.
 
```

Comment:
These exact contents are already in the configuration file. There's no need for this step. I think it is better to write the installation guide in as few steps as possible and then have another guide for configuration/installing on multiple hosts.